templates/clusterclass-dev-test.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+metadata:
+  name: dev-test
+spec:
+  controlPlane:
+    ref:
+      apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+      kind: KubeadmControlPlaneTemplate
+      name: ${CLUSTER_NAME}-control-plane-template
+    machineInfrastructure:
+      ref:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1alpha7
```

Comment:
the latest version is v1alpha8, let's use it please.

Reply:
Good point! I don't know how I missed that.
The v1alpha8 API is not in any release yet, so using it would require building your own image. However, with tilt that part is automatically done for you anyway, and the whole idea was to have something for development so I agree we should use v1alpha8 :slightly_smiling_face:

Reply:
This is what I hinted at when I said it is not part of a release. `clusterctl init` will not use the latest manifests and CRDs directly from the main branch since that would be risky and unstable. It will by default find the latest release and use that. For CAPO that means using v1alpha7.

However, for development that is not so useful since we need to test the latest changes that we are working on. Tilt is already configured to take the latest code and CRDs so if you use that, v1alpha8 will work. If you want to do it with clusterctl init, you need to [use local artifacts](https://cluster-api.sigs.k8s.io/clusterctl/developers#use-local-artifacts). I may be good to mention this, but I would not spend too much time on it. Just link to the CAPI docs for how to do it.

Reply:
Yeah I agree with Lennart here, let's focus on how to use tilt

Reply:
Okey the docs and everything else is now focusing on Tilt only.